zebra/zebra_vty.c

```diff
@@ -2674,7 +2674,7 @@ static void vty_show_ip_route_summary_prefix(struct vty *vty,
  * Show IPv6 mroute command.Used to dump
  * the Multicast routing table.
  */
-DEFUN (show_ipv6_mroute,
+/* DEFUN (show_ipv6_mroute,
```

Comment:
Need to check the history why is this implemented here? show ip mroute is in pim daemon not in zebra.